examples/template-with-jinja/README.md

```diff
@@ -0,0 +1,38 @@
+# Example with Jinja2 Template Engine
```

Comment:
I don't think the installation instructions are necessary, it is already covered in the readme of the project, although maybe not extensively enough: there's no info about the need to install `uvicorn`, `jinja2`, because I automatically assumed that if someone needs this package, they will know if they need to explicitly install those or not. Maybe it would be worth a mention in the readme.

I also don't think `requirements.txt` should be here. If there's a readme for the example (I think it is useful), I think it should describe the steps of creating a new project: create project folder, install dependencies, copy this code to your folder, run with uvicorn.

FastAPI's documentation (and many other projects) also has examples in this form.

Let me know what you think about this.

Reply:
> I don't think the installation instructions are necessary, it is already covered in the readme of the project, although maybe not extensively enough: there's no info about the need to install `uvicorn`, `jinja2`, because I automatically assumed that if someone needs this package, they will know if they need to explicitly install those or not. Maybe it would be worth a mention in the readme.

You are totally right but I believe examples should be ready, "out-of-the-box", and fully ready with all the dependencies (requirements.txt) and instructions (the audience might not have poetry on their machines, so offering virtualenv instructions).

> I also don't think `requirements.txt` should be here. If there's a readme for the example (I think it is useful), I think it should describe the steps of creating a new project: create project folder, install dependencies, copy this code to your folder, run with uvicorn.

But the current README.md file has all these septs, create a project folder (clone the repo section), install dependencies (install dependencies with poetry/virtualenv), copy this code to your folder (clone the repo section), run with uvicorn (run the application section)

> FastAPI's documentation (and many other projects) also has examples in this form.

I can't argue with that.

> Let me know what you think about this.

As I said, I was respecting the Flask Admin examples structure. You don't need to do anything to see a live example, except follow the instructions that only require a command line interface. You don't even need to open a code editor.

I still can remove these parts if you want, I don't bother but I was only caring about the developer experience as a starter.

Reply:
I'm considering your recommendation.

How about removing `requirements.txt` and replacing it with `pip install fasthx fastapi[all]`? I'd also still skip the `venv` and `poetry` parts, because if you're using virtual environments or a dependency (why no Pipenv instructions for example?), you know what to do if you see `pip install` in a guide.

Reply:
> I'm considering your recommendation.
> 
> How about removing `requirements.txt` and replacing it with `pip install fasthx fastapi[all]`? I'd also still skip the `venv` and `poetry` parts, because if you're using virtual environments or a dependency (why no Pipenv instructions for example?), you know what to do if you see `pip install` in a guide.

One other motivation to keep the `venv` example, is to encourage people to use virtualenvs as a best practice.

I'll remove the "installation instructions" with the entire README.md file.

Reply:
I have removed `README.md` and `requirements.txt` as you requested.

Reply:
> One other motivation to keep the venv example, is to encourage people to use virtualenvs as a best practice.

That's a good goal, but it can be done with several tools (venv, pyenv, poetry, pipenv, microvenv).

Thanks!